terraform-modules/aws/vpc/variables.tf

```diff
@@ -172,3 +172,20 @@ variable "default_security_group_tags" {
   default = {}
 }
 
+variable "map_public_ip_on_launch" {
```

Comment:
This field is mandatory in the new terrarform version, it was true by default before, We need false due we don't want to assign public ips in the instances.